src/styles/index.scss

```diff
@@ -1,3 +1,70 @@
 body {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  width: 706px;
+  padding: 0 10px;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__day {
+    align-content: center;
+    box-sizing: border-box;
+    height: 100px;
+    width: 100px;
+    color: black;
+    background-color: #eee;
+    border: 1px solid black;
+    text-align: center;
+    cursor: pointer;
+    transition-duration: 0.5s;
+
+    &:hover {
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      transition-duration: 0.5s;
+    }
+
+    @for $day from 1 through 31 {
+      &:nth-child(#{$day})::before {
+        content: '#{$day}';
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+      }
+    }
+
+    &:first-child {
+      margin-left: 606px;
```

Comment:
The 'margin-left' property is set to a fixed value of 606px for the first child. This might cause layout issues on smaller screens. Consider using a more responsive approach, such as percentages or media queries, to handle different screen sizes.